config/schedule.yml

```diff
@@ -58,9 +58,14 @@ remove_stale_vacancies:
   class: 'RemoveStaleVacanciesJob'
   queue: low
 
-remove_vacancies_that_expired_yesterday:
+remove_vacancies_that_expired_yesterday_from_algolia:
   cron: '0 03 * * *'
-  class: 'RemoveVacanciesThatExpiredYesterday'
+  class: 'RemoveVacanciesThatExpiredYesterdayFromAlgoliaJob'
+  queue: low
+
+remove_vacancies_that_expired_yesterday_from_google_index:
+  crom: '* 04 * * *'
```

Comment:
crontab.guru says this means "at every minute past hour 4" which I take to mean "at 4:00, 4:01, 4:02, 4:03, etc".

Seems a bit eager.

Reply:
Also, you have written the perfectly cromulent word `crom`.

Reply:
```suggestion
  cron: '0 04 * * *'
```

Reply:
Did sidekiq_spec.rb correctly fail due to the misspelling of `cron`?

Reply:
Hmm, seems like it didn't.

Reply:
We could change sidekiq_spec so that it catches this next time:

```ruby
# line 11
  let(:scheduled_jobs) { YAML.load_file("./config/schedule.yml").select { |_, v| v["cron"].present? }.map { |_, v| v["class"] } }
```

Reply:
I may make this change in another small PR? How do you feel about that?

Reply:
I assume you mean the sidekiq spec - in which case yes.